app/reducers/allowed.ts

```diff
@@ -1,4 +1,5 @@
 import { Meta } from '../actions/ActionTypes';
+import type { AnyAction, Reducer } from '@reduxjs/toolkit';
```

Comment:
I see that you're mixing between importing `AnyAction` from `redux` and `@reduxjs/toolkit`. Probably doesn't matter, but still nice to be consistent?